generators/generator-base.js

```diff
@@ -2454,7 +2454,7 @@ module.exports = class extends PrivateBase {
      */
     getAllJhipsterConfig(generator = this, force) {
         let configuration = generator.config.getAll() || {};
-        if ((force || !configuration.baseName) && jhiCore.FileUtils.doesFileExist('.yo-rc.json')) {
+        if (!configuration.baseName && jhiCore.FileUtils.doesFileExist('.yo-rc.json')) {
```

Comment:
Without this change, some configurations are undefined when using blueprint, as it will use configuration from `generator-jhipster` instead of `generator-jhipster-helloworld`

Not sure if the `.yo-rc.json` is correct:

```yaml
{
  "generator-jhipster-helloworld": {
    "promptValues": {
      "packageName": "com.mycompany.myapp"
    },
    "jhipsterVersion": "5.1.0",
    "applicationType": "monolith",
    "baseName": "jhipster",
    "packageName": "com.mycompany.myapp",
    "packageFolder": "com/mycompany/myapp",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "cacheProvider": "ehcache",
    "enableHibernateCache": true,
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "mysql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "60cdf1f2fc250f90434dc2865264445c4f6cef16",
    "clientFramework": "angularX",
    "useSass": false,
    "clientPackageManager": "yarn"
  },
  "generator-jhipster": {
    "promptValues": {
      "nativeLanguage": "en"
    },
    "jhipsterVersion": "5.1.0",
    "applicationType": "monolith",
    "baseName": "jhipster",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "enableTranslation": true,
    "clientPackageManager": "yarn",
    "nativeLanguage": "en",
    "languages": [
      "en"
    ],
    "blueprint": "generator-jhipster-helloworld"
  }
}
```

Reply:
actually, both values are needed,  so I think we can revert this change since I fixed this in a different way see https://github.com/jhipster/generator-jhipster/commit/9b6e29156885fa9e6e078e48e69d056ad390371d